avm/res/sql/server/database/main.bicep

```diff
@@ -0,0 +1,268 @@
+metadata name = 'SQL Server Database'
+metadata description = 'This module deploys an Azure SQL Server Database.'
+metadata owner = 'Azure/module-maintainers'
+
+@description('Required. The name of the database.')
+param name string
+
+@description('Conditional. The name of the parent SQL Server. Required if the template is used in a standalone deployment.')
+param serverName string
+
+@description('Optional. The collation of the database.')
+param collation string = 'SQL_Latin1_General_CP1_CI_AS'
+
+@description('Optional. The skuTier or edition of the particular SKU.')
+param skuTier string = 'GeneralPurpose'
+
+@description('Optional. The name of the SKU.')
+param skuName string = 'GP_Gen5_2'
+
+@description('Optional. Capacity of the particular SKU.')
+param skuCapacity int = -1
```

Comment:
Please change the implamentation to use nullable. To do so, you can make this parameter nullable, update the place where it is used to not check for `-1` and also update the reference in the parent

Reply:
It's in fact one of the biggest upsides for nullable

Reply:
done: https://github.com/Azure/bicep-registry-modules/pull/637/commits/373c68d9d04b531aa3673ea250d4b94c035c4dca

Reply:
done: https://github.com/Azure/bicep-registry-modules/pull/637/commits/373c68d9d04b531aa3673ea250d4b94c035c4dca and https://github.com/Azure/bicep-registry-modules/pull/637/commits/116cd0c7ca945bcc8d0714bfe432a7833f46f305